containers/fedora/user-data

```diff
@@ -22,7 +22,7 @@ write_files:
         WantedBy=cloud-init.service
 runcmd:
   - systemctl daemon-reload
-  - dnf install -y tcpdump qemu-guest-agent iperf3 dmidecode nginx lldpad kernel-modules nmap dhcp-server dhcp-relay sshpass podman ethtool libibverbs dpdk stress-ng iotop fio
+  - dnf install -y tcpdump qemu-guest-agent iperf3 dmidecode nginx lldpad kernel-modules nmap dhcp-server dhcp-relay sshpass podman ethtool libibverbs dpdk stress-ng iotop fio --skip-unavailable
```

Comment:
I'm not sure it is a good idea to add this param globally
if in future some of these packages will disappear in x86 image - better to know that during installation instead of debugging the failed test

Reply:
Hello @dshchedr, what do you thing of something like:
```
  - |
    PACKAGES="tcpdump qemu-guest-agent iperf3 dmidecode nginx lldpad kernel-modules nmap dhcp-server dhcp-relay sshpass podman ethtool libibverbs dpdk stress-ng iotop fio"
    ARCH=$(uname -m)
    if [ "${ARCH}" = "s390x" ]; then
      PACKAGES="tcpdump qemu-guest-agent iperf3 nginx lldpad kernel-modules nmap dhcp-server dhcp-relay sshpass podman ethtool libibverbs stress-ng iotop fio"
    fi
    dnf install -y ${PACKAGES}
  - dnf autoremove
  - dnf clean all
```

Reply:
@dshchedr I modified it according to the previously suggested approach, which means that "--skip-unavailable" is no longer necessary. What do you think?